fetchmail_inbox_invoice/__openerp__.py

```diff
@@ -0,0 +1,41 @@
+# coding: utf-8
+# Copyright (C) 2014 Therp BV (<http://therp.nl>)
```

Comment:
Thanks for the review! I don't think this deserves copyright, but I accepted a mention under contributors in the README.